### ISSUE TITLE
🐛 New government work email field missing in admin user profile

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#12168]
## 🐛 Bug

While logged in as a role that has access to view user profiles (other than yourself), the new government work email field is missing under the "Government information" section.

## 🦋 Expected Behaviour

All available information is displayed under the government information section of the admin user profile page.

## 📋 Steps to Reproduce

1. Find a user that has a work email set in the database
2. Login as admin `admin@test.com`
3. Navigate to that users profile `/admin/users/{userId}/profile
4. Scroll down to the "Government information" section
5. Observe the email field is not displayed

## 📸 Screenshot

![2024-12-02_14-21](https://github.com/user-attachments/assets/41a55943-522c-484c-8015-f815bb966fe9)

## 🙋‍♀️ Proposed Solution

Add missing fields to the display of this section.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/apps/web/src/components/UserProfile/ProfileSections/GovernmentInformationSection.tsx

## ✅ Acceptance Criteria

- [ ] All government info fields are displayed to roles that have access to them

**Comment by tristan-orourke** [comment]
@Jerryescandon do you want to add this to figma as well?

**Comment by Jerryescandon** [comment]
Yeah, I actually updated the figma files for this other one https://github.com/GCTC-NTGC/gc-digital-talent/issues/12140 but it's the same thing
